src/Expensify.js

```diff
@@ -19,6 +19,36 @@ import {growlRef} from './libs/Growl';
 import StartupTimer from './libs/StartupTimer';
 import {setRedirectToWorkspaceNewAfterSignIn} from './libs/actions/Session';
 import {create} from './libs/actions/Policy';
+import CONST from './CONST';
+import Log from './libs/Log';
+import listenToStorageEvents from './libs/listenToStorageEvents';
+
+// Initialize the store when the app loads for the first time
+Onyx.init({
+    keys: ONYXKEYS,
+    safeEvictionKeys: [ONYXKEYS.COLLECTION.REPORT_ACTIONS],
+    initialKeyStates: {
+
+        // Clear any loading and error messages so they do not appear on app startup
+        [ONYXKEYS.SESSION]: {loading: false, shouldShowComposeInput: true},
+        [ONYXKEYS.ACCOUNT]: CONST.DEFAULT_ACCOUNT_DATA,
+        [ONYXKEYS.NETWORK]: {isOffline: false},
+        [ONYXKEYS.IOU]: {
+            loading: false, error: false, creatingIOUTransaction: false, isRetrievingCurrency: false,
+        },
+        [ONYXKEYS.IS_SIDEBAR_LOADED]: false,
+    },
+    registerStorageEventListener: (onStorageEvent) => {
+        listenToStorageEvents(onStorageEvent);
+    },
+});
```

Comment:
Heads up this logic was moved to another file which is why it's showing as an addition. Probably we don't want to call it in both places.

https://github.com/Expensify/App/blob/936828848f2b1bd22602e87fbad3f2d6d029daa0/src/components/OnyxProvider.js#L11-L14

Reply:
Oh... that's why I had to re-add the imports 😄 
Removing.